config/vufind/config.ini

```
@@ -459,6 +459,9 @@ title_level_holds_mode = "disabled"
 ; can extend/override the VuFind\View\Helper\Root\Holdings::holdingIsVisible method.
 ;display_items_without_barcodes = true
 
+; Whether to display the loan type on items in the Holdings display.
```

Maybe worth a clarifying note here:

```suggestion
; Whether to display the loan type on items in the Holdings display (if supported by driver).
```